### PR TITLE
README: added ruby test commands

### PR DESCRIPTION
**Pull request by TamsilAmani**
### Description

Added commands to run tests in ruby with bazel

### Motivation and Context

Changes suggested by @titusfortner and @AutomatedTester 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by p0deje — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -209,6 +209,19 @@ To build the Ruby code run:
 ```sh
 bazel build //rb/...
 ```
+
+To run tests on a browser:
+
+```sh
+bazel test --cache_test_results=no --test_output=all //rb:<browsername>-test
```
Comment (titusfortner) [inline README.md:222]: I don't think this is useful without the list of available browsernames. Let's add:

* chrome
* firefox
* firefox-nightly (requires setting ENV["FIREFOX_NIGHTLY_BINARY"])
* safari
* safari-preview
* ie
* edge

File: README.md [hunk 2]
```diff
@@ -209,6 +209,19 @@ To build the Ruby code run:
...
+To run tests on a browser:
+
+```sh
+bazel test --cache_test_results=no --test_output=all //rb:<browsername>-test 
+```
+
+To run a particular test on a browser:
+
+```sh
+bazel test --cache_test_results=no --test_output=all --test_arg="-e<Test_Target>" //rb:<browsername>-test
```
Comment (titusfortner) [inline README.md:234]: Is there a list of Test Targets? How do I know what I can put here?

**Review by titusfortner — CHANGES_REQUESTED** [review]
I don't think this is enough info to help users directly. Would also like for us to add `//rb:unit-test` and ideally also: `//rb:remote-<browsername>-test`

File: README.md (hunk 1 above)
Comment (TamsilAmani) [reply README.md:222]: I was thinking to add the list but seeing other bindings not doing it, I refrained. Will add them

File: README.md (hunk 2 above)
Comment (TamsilAmani) [reply README.md:234]: `Test_Target` is not a list but a placeholder, just like `browsername`, for the specific test the user wants to run. You can see in line 224 I have given an example. 
`Example, Test_Target = Selenium::WebDriver::Devtools`

So if a contributor adds tests and wants to run it without running the whole test suite, he can put his test in `Test_Target` 

We can use a different name like `test_to_run`, or similar if you think it's better.

File: README.md (hunk 1 above)
Comment (titusfortner) [reply README.md:222]: Yeah, I think we should be doing it for all the bindings; we need to make bazel usage as obvious as possible.

File: README.md (hunk 2 above)
Comment (titusfortner) [reply README.md:234]: Right, I'm looking for examples again, what else besides `Selenium::WebDriver::Devtools` works?

Comment (TamsilAmani) [reply README.md:234]: We can have:
* Selenium::WebDriver::TakesScreenshot
* Selenium::WebDriver::Timeouts
* Selenium::WebDriver::Chrome::Driver
* Selenium::WebDriver::Firefox::Profile
* Selenium::Webdriver::Remote::Driver
etc...

Would you like me to add these to the README?

Comment (titusfortner) [reply README.md:234]: yes, listing a few examples is good. Easy to see what else can be used from that. thanks.

Comment (TamsilAmani) [reply README.md:234]: Changes incorporated.

File: README.md (hunk 1 above)
Comment (TamsilAmani) [reply README.md:222]: Made the relevant changes. Please have a look.

File: README.md [hunk 3]
```diff
@@ -210,18 +210,50 @@ To build the Ruby code run:
...
 To run tests on a browser:
 
 ```sh
 bazel test --cache_test_results=no --test_output=all //rb:<browsername>-test 
 ```
+_browsername_: 
+* chrome
+* edge
+* firefox
+* firefox-nightly
```
Comment (titusfortner) [inline README.md]:
```suggestion
* firefox-nightly (requires setting `ENV['FIREFOX_NIGHTLY_BINARY']` with the path to the browser executable)
```

**Review by titusfortner — APPROVED** [review]

**Pull request merged**